doc/sphinx/proof-engine/tactics.rst

```diff
@@ -1257,115 +1257,87 @@ Managing the local context
 
-   .. tacv:: clear
-
-      This variants clears all the hypotheses except the ones the goal depends on.
-
-   .. tacv:: clear dependent @ident
-
-      This clears the hypothesis :token:`ident` and all the hypotheses that
-      depend on it.
-
-   .. tacv:: clearbody {+ @ident}
-      :name: clearbody
-
-      This tactic expects :n:`{+ @ident}` to be local definitions and clears
-      their respective bodies.
+      This tactic expects :n:`{+ @ident}` to be :term:`local definitions <context-local definition>`
+      and clears their respective bodies.
       In other words, it turns the given definitions into assumptions.
 
       .. exn:: @ident is not a local definition.
          :undocumented:
 
-.. tacn:: revert {+ @ident}
-   :name: revert
+.. tacn:: revert {+ @ident }
 
-   This applies to any goal with variables :n:`{+ @ident}`. It moves the hypotheses
-   (possibly defined) to the goal, if this respects dependencies. This tactic is
+   Moves the specified hypotheses
+   (possibly defined) to the goal, if this respects dependencies. This is
    the inverse of :tacn:`intro`.
 
-   .. exn:: No such hypothesis.
-      :undocumented:
-
    .. exn:: @ident__1 is used in the hypothesis @ident__2.
       :undocumented:
 
-   .. tacv:: revert dependent @ident
-      :name: revert dependent
-
-      This moves to the goal the hypothesis :token:`ident` and all the
-      hypotheses that depend on it.
-
-.. tacn:: move @ident__1 after @ident__2
-
-   This moves the hypothesis named :n:`@ident__1` in the local context after
-   the hypothesis named :n:`@ident__2`, where “after” is in reference to the
-   direction of the move. The proof term is not changed.
-
-   If :n:`@ident__1` comes before :n:`@ident__2` in the order of dependencies,
-   then all the hypotheses between :n:`@ident__1` and :n:`@ident__2` that
-   (possibly indirectly) depend on :n:`@ident__1` are moved too, and all of
-   them are thus moved after :n:`@ident__2` in the order of dependencies.
-
-   If :n:`@ident__1` comes after :n:`@ident__2` in the order of dependencies,
-   then all the hypotheses between :n:`@ident__1` and :n:`@ident__2` that
-   (possibly indirectly) occur in the type of :n:`@ident__1` are moved too,
-   and all of them are thus moved before :n:`@ident__2` in the order of
-   dependencies.
+   .. tacn:: revert dependent @ident
 
-   .. tacv:: move @ident__1 before @ident__2
-      :name: move … before …
+      Moves the named hypothesis and all the hypotheses that depend on it to the goal.
 
-      This moves :n:`@ident__1` towards and just before the hypothesis
-      named :n:`@ident__2`.  As for :tacn:`move`, dependencies
-      over :n:`@ident__1` (when :n:`@ident__1` comes before :n:`@ident__2` in
-      the order of dependencies) or in the type of :n:`@ident__1`
-      (when :n:`@ident__1` comes after :n:`@ident__2` in the order of
-      dependencies) are moved too.
+      .. todo PR: is this the inverse of :tacn:`clear dependent`?
```

Comment:
No. It's still the inverse of `intros` but with a very specific list of arguments.